cmd/internal/format.go

```diff
@@ -154,13 +155,15 @@ func writeCountriesTable(countries *openaq.CountriesResponse, headers []string)
 	var columns = len(headers)
 	tw := table.NewWriter()
 	writeTableHeader(tw, headers)
+	fmt.Println(countries.Results[0].Parameters)
 	for _, s := range countries.Results {
 		row := make(table.Row, 0, columns)
 		row = append(row, strconv.FormatInt(s.ID, 10))
 		row = append(row, s.Code)
 		row = append(row, s.Name)
 		row = append(row, s.DatetimeFirst.Format(time.RFC3339))
 		row = append(row, s.DatetimeLast.Format(time.RFC3339))
+		row = append(row, joinParamDisplayNames(s.Parameters))
 		row = append(row, "")
```

Comment:
this column need to be removed i think, there are 9 header columns but now 10 data columns so i think providers_count is getting pushed off the table.